app/models/bike.rb

```diff
@@ -1,2 +1,9 @@
 class Bike < ApplicationRecord
+  has_many :bookings
+  has_many :users, through: :users
```

Comment:
Here rel to user should rather be...
belongs_to :user
because the user has already has_many :bikes